### PR TITLE
docs: element updates

### LINKED ISSUE #1252: [docs]: Very minor updates to Elements page
@eyevana Here's the mock-up for the `All elements` page which includes `Table`.

https://www.figma.com/proto/Tx1R9SMk6b0dCSJ84Xi0SN/All-elements?page-id=0%3A1&type=design&node-id=1-2&viewport=74%2C98%2C1&t=AvkEF8orieVbhmPS-1&scaling=min-zoom&mode=design

Folder of images:

https://www.dropbox.com/sh/vlpos6ghlv1oabj/AACpCj-9gKQVDXY3o8mvxK5aa?dl=0

Page changes:

- [x] Every element link goes to some other page besides `Overview`, please fix that
- [x] Replace `Alert` image
- [x] Replace `Audio player` image
- [x] Change `Audio Player` to `Audio player` (sentence case)
- [x] Replace `Avatar` image
- [x] Change `Avatar` description to `Visually represents a user in a masthead or navigation`
- [x] Replace `Badge` image
- [x] Replace `Blockquote` image
- [x] Change `Code Block` to `Code block` (sentence case)
- [x] Change `Code block` description to `Formats code strings within a container`
- [x] Replace `Footer` image
- [x] Replace `Navigation (primary)` image
- [x] Replace `Pagination` image
- [x] Change `Pagination` file name from `all-elements-nav-pagination.png` to `all-elements-pagination.png`
- [x] Change `Spinner` file name from `all-elements-nav-spinner.png` to `all-elements-spinner.png`
- [x] Replace `Subnavigation` image
- [x] New image for `Table` is in the Dropbox

Thanks!

### PR DESCRIPTION
**Pull request by eyevana**
## What I did

Detailed list of changes in issue #1252.

1. Updated element screenshots and summaries 
2. Created filter to transform titles to sentence case
3. Fixed all links on elements page; should be going to overview tab by default

Closes #1252

### CODE REVIEW

**Comment by coreyvickery** [comment]
@eyevana I'm not seeing `Table` or is that addition in the other issue?

**Comment by coreyvickery** [comment]
> @eyevana I'm not seeing `Table` or is that addition in the other issue?

Nevermind, it is in the other PR.

**Comment by coreyvickery** [comment]
@eyevana

Tiny final changes:

- [ ] Change h1 to `All elements`
- [ ] Remove period in the `Avatar` description
- [ ] Remove period in the `Code block` description
- [ ] Change `Navigation (secondary)` description to `Propagates related content across a series of pages`

You can merge after these are done, I don't need to approve a second time.

File: elements/rh-avatar/rh-avatar.ts
Comment (bennypowers) [inline elements/rh-avatar/rh-avatar.ts:1]: patch changeset for fixing docs

File: elements/rh-code-block/rh-code-block.ts
Comment (bennypowers) [inline elements/rh-code-block/rh-code-block.ts:1]: patch changeset for fixing docs

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: .changeset/loud-dolphins-enjoy.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+Updated all elements docs page with new summaries, images, and correct links
```
Comment (bennypowers) [inline .changeset/loud-dolphins-enjoy.md:5]: as far as the changeset is concerned (i.e. the npm package), the docs pages are not important, rather, the changes to the custom-elements.json file are what are important here.
```suggestion
`<rh-avatar>`:
`<rh-code-block>`: Updated custom elements manifest summaries
```

**Review by bennypowers — APPROVED** [review]

**Pull request merged**